marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/smart-mastering/match-and-merge/asymmetric-match-merge-notify.xqy

```diff
@@ -0,0 +1,105 @@
+xquery version "1.0-ml";
+import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
+hub-test:reset-hub()
+
+;
+
+xquery version "1.0-ml";
```

Comment:
Oof - you shouldn't have to do this here. Keep it for now, I'll make a PR to support modules via hub-test:load* (or if you want to knock it out in a separate PR and beat me to it, no problem!).

Reply:
I wanted a clean slate for this test with no side effects from setup,xqy, suite-setup,xqy or any other test-data docs. Without creating a new test suite, I did what I had to do.